apps/camera/locales/camera.en-US.properties

```diff
@@ -21,6 +21,7 @@ close-button  = Close
 
 delete-photo? = Delete photo?
 delete-video? = Delete video?
+gaia-menu-cancel = Cancel
```

Comment:
Since this is a blocker, it would be better to include the localization file from shared/ so we don't have late l10n changes here. You can see how to do this here: https://github.com/mozilla-b2g/gaia/blob/master/apps/video/index.html#L11